content/concepts/plugins.md

```diff
@@ -1,5 +1,84 @@
 ---
 title: Plugins
 sort: 3
+contributors:
+  - TheLarkInn
 ---
-> plugins configuration option
+
+**Plugins** are the [backbone]('https://github.com/webpack/tapable') of webpack. webpack itself is built on the **same Plugin system** that you use in your webpack configuration!! 
+
+They also serve the purpose of doing **anything else** that a [loader]('./loaders') cannot do.
+
+## Anatomy
+
+A webpack **Plugin** is a JavaScript object that has an `apply` property. This `apply` property is called by the webpack compiler, giving access to the **entire** compilation lifecycle.
+
+
+**ConsoleLogOnBuildWebpackPlugin.js**
+
+```javascript
+function ConsoleLogOnBuildWebpackPlugin() {
+  
+};
+
+ConsoleLogOnBuildWebpackPlugin.prototype.apply = function(compiler) {
```

Comment:
I guess specific info like this would fit below `content/plugins` better. It's fine to show a basic example here. Still good to cover somewhere perhaps.